rewrite_external_links/middleware.py

```diff
@@ -26,22 +26,22 @@ class RewriteExternalLinksMiddleware(object):
         (?P<link>https?://{}[^'">]*)  # href link
         (?P<after>[^>]*)  # content after the href attribute to the closing bracket `>`
     '''.format(safe_urls), re.VERBOSE)
-    external_link_root = reverse('external_link')
 
     def process_response(self, request, response):
+        external_link_root = reverse('external_link')
```

Comment:
I couldn't use `reverse_lazy` as I get something else than a string when I pass it to `startswith`.